cmd/erasure-healing.go

```diff
@@ -256,6 +249,21 @@ func (er erasureObjects) healObject(ctx context.Context, bucket string, object s
 		DataBlocks:   len(storageDisks) - er.defaultParityCount,
 	}
 
+	if !opts.NoLock && newObjectLayerFn() != nil {
```

Comment:
```suggestion
	if !opts.NoLock && newObjectLayerFn() != nil {
```

If the code comes here `objectLayer` is initialized why have this? should be sufficient to just do? 


```suggestion
	if !opts.NoLock {
```